pom.xml

```diff
@@ -304,7 +304,6 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <reuseForks>false</reuseForks>
-                    <forkCount>${concurrency}</forkCount>
                 </configuration>
```

Comment:
Is this required?

Reply:
it prevented that the forkCount can be set by the ci and that you can set it via command line with -DforkCount ro othat the forkCount setting from line 68 was effective